aspnetcore/index.md

```diff
@@ -16,45 +16,52 @@ uid: index
 
 By [Daniel Roth](https://github.com/danroth27), [Rick Anderson](https://twitter.com/RickAndMSFT), and [Shaun Luttin](https://twitter.com/dicshaunary)
 
-ASP.NET Core is a new [open-source](https://github.com/aspnet/home) and cross-platform framework for building modern, cloud-based, Internet-connected applications, such as web apps, IoT apps and mobile backends. It was architected to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run ASP.NET Core apps on Windows, Mac and Linux. ASP.NET Core apps can [run on .NET Core or on the .NET Framework](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server).
+ASP.NET Core is a cross-platform, high-performance, [open-source](https://github.com/aspnet/home) framework for building modern, cloud-based, Internet-connected applications. With ASP.NET Core, you can:
 
-## Why use ASP.NET Core?
+* Build web apps and services, IoT apps, and mobile backends.
```

Comment:
I think this first set of bullet points should tell me what ASP.NET Core is and what it should be used for. Some of these bullet points don't seem on topic (like using your favorite dev tools xplat). I think we need to iterate on this list. Maybe it should be a separate discussion though